src/app/backend/integration/api/types.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// IntegrationID is a unique identification string that every integrated app has to provide.
```

Comment:
Can you provide in-comment example of what an id would be? And where I can find other IDs to avoid conflicts.